inst/htmlwidgets/Widget_FlagOverTime.js

```diff
@@ -8,10 +8,34 @@ HTMLWidgets.widget({
 
     // Apply styles to make the content scrollable if it gets too long
     el.style.overflowY = 'auto';  // Enable vertical scrolling
+    el.style.overflowX = 'hidden';  // Prevent horizontal scrolling
     el.style.maxHeight = '400px'; // Set the max height for the widget
+    el.style.width = '100%';        // Set width to 100% to prevent overflow
 
     return {
       renderValue: function(x) {
+        // Create the toggle button
+        const toggleButton = document.createElement('button');
```

Comment:
maybe try to implement the toggle switch, like we use in Report_StudyInfo.R (lines 37-44)

Reply:
I think I tried to go somewhere along those lines of just doing it solely in R but I couldn't get past the fact the js widget at its current state only takes 1 table to display, not sure how it would work otherwise @samussiah any thoughts?

Reply:
yeah, at the moment its just a default styled button, but it works as expected. elsewhere this toggle button is inserted via html, so i'm not clever enough to quickly figure out how to do that here in js. we can add another issue if folks are annoyed by how it looks.

Reply:
i actually was able to get like 80% of the way there- got the switch to appear with the label next to it, but it doesn't slide when clicked. i think it's probably worth figuring this out tomorrow before merging this in. @jonthegeek or @samussiah - any tips? 
![image](https://github.com/user-attachments/assets/a2ec0443-f2f9-44a2-abf8-1a4a15014b50)

Reply:
got it to slide when clicked!! i'll let someone else take a look before i approve though